### PR TITLE
Working GCS bucket

### PR DESCRIPTION
**Pull request by likelymichael**
Tested with linux docker image that was able to access bucket using `gsutil`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,164 @@
...
+## Bundle
+
+<!-- COMPLIANCE:START -->
+
+Security and compliance scanning of our bundles is performed using [Bridgecrew](https://www.bridgecrew.cloud/). Massdriver also offers security and compliance scanning of operational infrastructure configured and deployed using the platform.
+
+| Benchmark                                                                                                                                                                                                                                                       | Description                        |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------- |
+| [![Infrastructure Security](https://www.bridgecrew.cloud/badges/github/massdriver-cloud/gcp-gcs-bucket-application-assets/general)](https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=&benchmark=INFRASTRUCTURE+SECURITY) | Infrastructure Security Compliance |
+
```
Comment (chrisghill) [inline README.md:37]: Remove this line

File: src/_providers.tf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  gcp_project_id = var.gcp_authentication.data.project_id
+}
+
+provider "google" {
+  project     = local.gcp_project_id
+  credentials = jsonencode(var.gcp_authentication.data)
+  region      = var.bucket.region
+}
+
+provider "google-beta" {
```
Comment (chrisghill) [inline src/_providers.tf:33]: you can remove this provider

File: src/_providers.tf [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  required_providers {
+    massdriver = {
+      source  = "massdriver-cloud/massdriver"
+      version = "~> 1.1"
+    }
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 4.0"
+    }
+    google-beta = {
```
Comment (chrisghill) [inline src/_providers.tf:12]: you can remove this provider

File: src/_providers.tf [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      source  = "hashicorp/google-beta"
+      version = "~> 4.0"
+    }
+    null = {
```
Comment (chrisghill) [inline src/_providers.tf:16]: you can remove this provider

**Review by chrisghill — CHANGES_REQUESTED** [review]
Some minor changes. Once made I'll approve.

Just to confirm, you tested both IAM policies and they work, correct?

**Comment by likelymichael** [comment]
> Some minor changes. Once made I'll approve.
> 
> Just to confirm, you tested both IAM policies and they work, correct?

Yes, I tested creating a .txt file, and then read from it

**Review by chrisghill — APPROVED** [review]

**Pull request merged**